GooglePlayInstant/Editor/PostBuildCommandLineDialog.cs

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using GooglePlayInstant.Editor.GooglePlayServices;
+using UnityEngine;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// A CommandLineDialog that waits to run the specified command until after an AppDomain reset.
+    ///
+    /// Several seconds after a call to BuildPlayer finishes, Unity reloads all Editor scripts and resets the
+    /// AppDomain, causing any running threads to be aborted. This EditorWindow waits to run the specified
+    /// command until after the AppDomain is reset.
+    /// </summary>
+    public class PostBuildCommandLineDialog : CommandLineDialog
+    {
+        /// <summary>
+        /// Struct encapsulating parameters for running a command line process.
+        ///
+        /// Note: supports a single environment variable.
+        /// </summary>
+        [Serializable]
+        public struct CommandLineParameters
+        {
+            [SerializeField] public string FileName;
+            [SerializeField] public string Arguments;
+            [SerializeField] public string EnvironmentKey;
+            [SerializeField] public string EnvironmentValue;
```

Comment:
it feels a little awkward to only allow a single EnvironmentKV override

Reply:
Done.